python/python/glide/async_commands/core.py

```diff
@@ -4119,3 +4124,28 @@ async def object_refcount(self, key: str) -> Optional[int]:
             Optional[int],
             await self._execute_command(RequestType.ObjectRefCount, [key]),
         )
+
+    async def flushall(self, flushmode: Optional[FlushMode] = None) -> str:
```

Comment:
I think python client has special type for `OK`. If yes, update code and doc
```suggestion
    async def flushall(self, flushmode: Optional[FlushMode] = None) -> TOK:
```

Reply:
in constants.py theres `TOK = Literal["OK"]` and `OK: str = "OK"`, why are there a str value and a str literal version? how do we decide which one to use

Reply:
They use both as I see, I don't know what is correct.
@aaron-congo is the python expert.